DataConnectors/CEF/cef_troubleshoot.py

```diff
@@ -450,6 +451,23 @@ def file_contains_string(file_tokens, file_path):
     return all(check_token(token, content) for token in file_tokens)
 
 
+def check_file_permissions(file_path, permissions=file_default_permissions):
+    # get the octal representation of the file permissions
+    get_permissions = subprocess.Popen(["stat", "-c", "'%a'", file_path], stdout=subprocess.PIPE)
+    o, e = get_permissions.communicate()
+    if e is not None:
+        print_warning("Unable to verify file permissions for path:" + file_path)
+        return False

```

Comment:
You aren't using the return value, maybe you can remove it.

Its a common issue in both files (with the old functions), some functions have a return value and some don't.
maybe we should pick a standard and stick with it.